### PR TITLE
FastDDS downgrade and add caml proxy

### PR DESCRIPTION
**Pull request by mehmetkillioglu**
## Changelog;
- The FastDDS version is downgraded to 2.10.1 due to SROS issues.
- PX4-msgs version fixed to 5.0.0* to prevent conflicts with rebase work
- camlproxyclient added to the baseimage. It is not going to be used by default. The old implementation of pkcs11-proxy1 will still functions as the relevant env variables points to that.

### CODE REVIEW

**Review by jari-hodju — APPROVED** [review]
Bit of a chicken and egg issue with fog-health.
Looks good to me.

File: Dockerfile [hunk 1]
```diff
@@ -52,6 +53,7 @@ RUN apt update -y \
         ros-core \
         tf2 \
         tf2-geometry-msgs \
+    && apt clean \
```
Comment (maseabunikie) [inline Dockerfile:56]: Just curiosity. What is the difference of having this command here?
Does it release more space in the image?

Comment (jari-hodju) [reply Dockerfile:56]: Yes. Removing /var/lib/apt/lists/* does not do all the cleaning. Using apt clean gives you ~7 MB smaller images.

Comment (maseabunikie) [reply Dockerfile:56]: Great! :)

**Pull request merged**